swc/cli_repositories.bzl

```diff
@@ -1,16 +1,17 @@
-"@generated by @aspect_rules_js//npm/private:npm_translate_lock.bzl from @aspect_rules_swc//swc:pnpm-lock.yaml"
+"@generated by @aspect_rules_js//npm/private:npm_translate_lock.bzl from //swc:pnpm-lock.yaml"
 
 load("@aspect_rules_js//npm:npm_import.bzl", "npm_import")
 
 def npm_repositories():
-    "Generated npm_import repository rules corresponding to npm packages in @aspect_rules_swc//swc:pnpm-lock.yaml"
+    "Generated npm_import repository rules corresponding to npm packages in //swc:pnpm-lock.yaml"
     npm_import(
         name = "swc_cli__at_nodelib_fs.scandir__2.1.5",
         root_package = "swc",
         link_workspace = "aspect_rules_swc",
         link_packages = {},
         package = "@nodelib/fs.scandir",
         version = "2.1.5",
+        lifecycle_hooks_no_sandbox = True,
```

Comment:
What made this required?

Reply:
This is the new default since running lifecycle hooks in the sandbox by default is a performance hit for no good reason

Reply:
(this is a generated file)